test/suite_test.go

```diff
@@ -66,6 +68,16 @@ func (e *e2e) SetupSuite() {
 		)
 		e.Nil(os.Chmod(e.kindPath, 0o755))
 	}
+
+	e.kubectlPath = filepath.Join(buildDir, "kubectl")
+	if !util.Exists(e.kubectlPath) {
+		e.run(
+			"curl", "-o", e.kubectlPath,
```

Comment:
When downloading and executing binaries, it is good practice to validate their checksum first. Here's an example:

`echo 5a99ce73a14da6ab561016b83906c398f082dd7571771b6b16430445f6668587a2ad2f5ca9e5a330bcc3fad5f4a7f21109042defcd674acae8cd7170b49ed947 ~/Downloads/kubectl | sha512sum -c -`

If the checkshum does not match it will return a non-zero exit code - which you can use to fail the setup. 

I would recommend that the checksum value is kept on a variable to be updated every time the kubectl version is changed. To find the correct checksum for a given version:

https://storage.googleapis.com/kubernetes-release/release/v1.18.5/bin/linux/amd64/kubectl.sha512

Reply:
Sure, I added a new SHA verification helper function and added it to the download.